languages/julia/exercises/concept/fibonacci-iterator/.docs/introduction.md

```diff
@@ -0,0 +1,112 @@
+## iterator-protocol
+
+<!-- TODO: Motivate why iterators are useful -->
+
+To implement the informal iteration interface, you need to define two methods `iterate(iter::MyIter)` and `iterate(iter::MyIter, state)`.
+Both methods must return a tuple of an item and the state of the iterator.
+The first method will return the first item and state, while the second method will return the next item and state.
+The iteration ends if `iterate` returns `nothing`.
+
+!!! note
+    Julia currently doesn't have a way to formally define interfaces.
+    To implement an interface, you need to look up which methods need to be defined in the Manual, or in the documentation of the package that "defines" the interface.
+    Alternatively, you can work your way through `MethodErrors` to find out which methods you need to implement:
+    ```julia
```

Comment:
It looks like this does not render correctly on GitHub. Do you need an empty line before this?

Reply:
Seems to be an issue with GitHub, locally it renders just fine. I'm guessing it will depend on how https://github.com/exercism/v3-website/issues/266 is implemented.

```suggestion
    <!-- TODO check if code blocks work within info boxes -->
    ```julia
```